pytorch_lightning/trainer/training_loop.py

```diff
@@ -71,8 +71,10 @@ def on_trainer_init(
         self.trainer.train_dataloader = None
         self.automatic_optimization = automatic_optimization
 
-        self.trainer.max_epochs = max_epochs
-        self.trainer.min_epochs = min_epochs
+        # If neither max_epochs or max_steps is set, then use existing default of max_epochs = 1000
+        self.trainer.max_epochs = 1000 if (max_epochs is None and max_steps is None) else max_epochs
```

Comment:
could be a constant

Reply:
Could be but no real need. You can set it with max_epochs

Reply:
I mean, it's not like in the ModelCheckpoint constants, where you'd need to hack it to change the last checkpoint name if SAVE_LAST_NAME didn't exist